lib/ckeditor/hooks/action_policy.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'action_policy'
+
+module Ckeditor
+  module Hooks
+    # This adapter is for the Pundit[https://github.com/elabs/pundit] authorization library.
+    # You can create another adapter for different authorization behavior, just be certain it
+    # responds to each of the public methods here.
+    class ActionPolicyAuthorization
+      include Ckeditor::Helpers::Controllers
+
+      # See the +authorize_with+ config method for where the initialization happens.
+      def initialize(controller)
+        @controller = controller
+        # use ckeditor_current_user instead of default current_user so it works with
+        # whatever current user method is defined with Ckeditor
+        @controller.class.authorize :user, through: :ckeditor_current_user
```

Comment:
We can avoid this passing [context explicitly](https://actionpolicy.evilmartians.io/#/authorization_context?id=explicit-context):

```rb
authorize!(model_object,  context: {user: ckeditor_current_user})
```